backend/src/packages/chats/chat.service.ts

```diff
@@ -32,6 +33,14 @@ class ChatService implements Service {
     return Promise.resolve(null);
   }
 
+  public async findById(
+    id: number,
+  ): Promise<ReturnType<ChatEntity['toObject']> | null> {
+    const chat = await this.chatRepository.findById(id);
+
+    return chat?.toObject() ?? null;
+  }
```

Comment:
do we actually need this?

Reply:
do you mean findById func? If yes we need this because without it we can`t check who is trying to delete this chat